spec/bundler/stub_specification_spec.rb

```diff
@@ -0,0 +1,23 @@
+# frozen_string_literal: true
+require "spec_helper"
+
+if Bundler.rubygems.provides?(">= 2.1")
+  RSpec.describe Bundler::StubSpecification do
+    let(:with_gem_stub_spec) do
+      stub = Gem::Specification.stubs.first
+      described_class.from_stub(stub)
+    end
+
+    let(:with_bundler_stub_spec) do
+      described_class.from_stub(with_gem_stub_spec)
+    end
+
+    describe "#to_spec" do
+      it "returns a Gem::Specification" do
+        bundle :install
```

Comment:
I was trying to address https://github.com/bundler/bundler/pull/5593#discussion_r112075326

I'm not familiar with the DSL used in the tests :/